### ISSUE TITLE
Emojis in CSS broken since 2.0.0

### ISSUE THREAD

**Issue by chrisEff** [webpack-contrib/css-loader#880]
* Operating System: macOS 10.14.2 (Mojave)
* Node Version: 10.14.2 (yes! it's by coincidence actually the same as the OS version 😄)
* NPM Version: 6.5.0
* webpack Version: 4.28.2
* css-loader Version: 2.0.0, 2.0.1, 2.1.0

### Expected Behavior

In my CSS I have a section with a `:before` selector and a `content` property. The `content` property contains a light bulb emoji. With css-loader 1.01 the emoji is converted into `\1F4A1`, which Chrome then in turn nicely renders as a lightbulb.

### Actual Behavior

With every version since 2.0.0 the emoji becomes `ð¡`, which does not get rendered as an emoji.

### Code

```css
.light.on .bulb:before{
	content: '💡';
}
```

```js
require('style-loader!css-loader!./style.css')
```

### How Do We Reproduce?

Use an emoji in a CSS `content` property and require the CSS file via css-loader + style-loader, like shown in the code snippets above. My webpack config doesn't contain anything that seems relevant for this.

**Comment by evilebottnawi** [comment]
@chrisEff we have tests on emoji and all work fine, please create minimum reproducible test repo

**Comment by evilebottnawi** [comment]
Tests https://github.com/webpack-contrib/css-loader/pull/881 and you can see emoji are not broken

**Comment by chrisEff** [comment]
Here you go:
https://github.com/chrisEff/css-loader-emoji-bug
Run `npm i` and `npx webpack` and then open the `public/index.html` file in a browser.
You will see some strange characters in front of "Test".

But if you change the css-loader version in `package.json` to `1.0.1` and do the same steps again, you'll see a nicely rendered light bulb.

**Comment by evilebottnawi** [comment]
@chrisEff weird, confirmed, investigate

**Comment by evilebottnawi** [comment]
@chrisEff no, it is not bug in `css-loader`. You need add `<meta charset="utf-8">` to fix it.

TL;DR

Without `<meta charset="utf-8">` you page interpreted based on characters in document (for me it is `windows-1252`), you can check this run `document.charset` in `console` so emoji broken. More information about this https://linuxwebdevelopment.com/how-to-insert-emojis-in-html/ , you can find you bug in `step 0`, thanks for issue

**Comment by chrisEff** [comment]
A big THANK YOU for investigating and coming up with with a solution so quickly! 👍 

But still: If this is the problem, why did it work then with the old css-loader?
